tests/Integration/Http/Controller/Admin/DashboardControllerTest.php

```diff
@@ -15,7 +15,7 @@
 
 use OpenCFP\Domain\Model\Talk;
 use OpenCFP\Test\Helper\RefreshDatabase;
-use OpenCFP\Test\WebTestCase;
```

Comment:
As far as I can tell, extending this `abstract` test class indicates an integration test.